pkg/subctl/cmd/gather/logs.go

```diff
@@ -120,3 +124,39 @@ func findPods(clientSet kubernetes.Interface, byLabelSelector string) (*corev1.P
 
 	return pods, nil
 }
+
+func getPreviousLogs(pod *corev1.Pod, podLogOptions *corev1.PodLogOptions, info Info) error {
+	podLogOptions.Previous = true
+	logRequest := info.ClientSet.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, podLogOptions)
+	logStream, _ := logRequest.Stream()
+
+	// TODO: Check for error other than "no previous pods found"
```

Comment:
yeah we shouldn't ignore the error.

Reply:
What about the TODO here? It's not a big deal - I guess we can deal with it later.